discord/asset.py

```diff
@@ -346,6 +346,15 @@ def _from_user_banner(cls, state: _State, user_id: int, banner_hash: str) -> Sel
             animated=animated,
         )
 
+    @classmethod
+    def _from_primary_guild(cls, state: _State, guild_id: int, icon_hash: str) -> Self:
+        return cls(
+            state,
+            url=f'{cls.BASE}/clan-badges/{guild_id}/{icon_hash}.png?size=16',
```

Comment:
```suggestion
            url=f'{cls.BASE}/clan-badges/{guild_id}/{icon_hash}.png?size=64',
```
16x16 is really tiny for a default xd